ROMFS/px4fmu_common/init.d-posix/1016_iris_rtps.post

```diff
@@ -1,2 +1,4 @@
+#!/bin/sh
 # shellcheck disable=SC2154
-micrortps_client start -t UDP
+
+micrortps_client start -t UDP -r $((2019+2*px4_instance)) -s $((2020+2*px4_instance))
```

Comment:
Where does these port numberings come from?

All other ports are in the range of `1454*`, `1857*`. Wouldn't make sense we set the port range more consistent to what is being used?

Reply:
This port number is the default number for "micrortps" module.

Reply:
Okay, in that case makes sense